drivers/dai/intel/ssp/ssp.c

```diff
@@ -2580,6 +2586,11 @@ static void ssp_release_ip(struct dai_intel_ssp *dp)
 
 		dai_ssp_post_stop(dp);
 
+#if SSP_IP_VER >= SSP_IP_VER_2_0
+		sys_write32(DAI_INTEL_SSP_CLOCK_XTAL_OSCILLATOR,
+			    dai_i2svss_base(dp) + I2SLCTL_OFFSET);
```

Comment:
this is equivalent to `sys_write32(0, dai_i2svss_base(dp) + I2SLCTL_OFFSET)`, do you want to reset the whole register or only the MLCS bits like at line 1994?

Reply:
ah right, just the clock bits, updated